readme.md

```diff
@@ -934,7 +934,7 @@ A function that transform [`Response`](#response) into an array of items. This i
 Type: `Function`\
 Default: [`Link` header logic](source/index.ts)
 
-The function takes three arguments:
+The function takes an object with the following properties:
 - `response` - The current response object.
 - `allItems` - An array of the emitted items.
```

Comment:
Maybe add a reference to `pagination.stackAllItems` behaviour?

Reply:
It's just a scroll away: https://github.com/sindresorhus/got#paginationstackallitems